packages/react-components/src/shared/src/useMergedRefs.ts

```diff
@@ -11,7 +11,7 @@ export function assignRef<T>(ref: ForwardedRef<T>, node: T): void {
     }
 }
 
-type MergedRef<T> = ((instance: T | null) => void) & MutableRefObject<T | null>;
+export type MergedRef<T> = ((instance: T | null) => void) & MutableRefObject<T | null>;
```

Comment:
Can use the RefCallback type instead? The idea of mergeRefs is really to mimick a ref callback behavior the same signature